libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/navigation/speedlimit/SpeedLimitWidget.kt

```diff
@@ -6,163 +6,60 @@ import android.graphics.Color
 import android.graphics.Paint
 import android.graphics.Rect
 import android.graphics.RectF
-import com.mapbox.androidauto.car.map.widgets.ImageOverlayHost
-import com.mapbox.androidauto.car.map.widgets.Margin
-import com.mapbox.androidauto.car.map.widgets.WidgetPosition
 import com.mapbox.androidauto.logAndroidAuto
 import com.mapbox.maps.EdgeInsets
+import com.mapbox.maps.MapboxExperimental
+import com.mapbox.maps.renderer.widget.BitmapWidget
+import com.mapbox.maps.renderer.widget.WidgetPosition
 
 /**
  * Widget to display a speed limit sign on the map.
  */
+@OptIn(MapboxExperimental::class)
 class SpeedLimitWidget(
     /**
      * The position of the widget.
      */
-    widgetPosition: WidgetPosition = WidgetPosition.BOTTOM_RIGHT,
+    private val position: WidgetPosition = WidgetPosition(
+        vertical = WidgetPosition.Vertical.BOTTOM,
+        horizontal = WidgetPosition.Horizontal.RIGHT,
+    ),
     /**
-     * The left margin of the widget relative to the map.
+     * The horizontal margin of the widget relative to the map.
      */
-    private val marginLeft: Float = 10f,
+    marginX: Float = 26f,
```

Comment:
Is this specific to screen resolutions?

What happens when you run this on a different resolution https://github.com/mapbox/mapbox-navigation-android-examples/issues/81#issuecomment-1085179008

Reply:
If there is an issue there, the issue would be existing. So if you test it please take some screenshots and post them on a new issue

Reply:
Turns out, testing other resolutions still takes a few steps. am posting the steps just because we're talking about it

1. Update to a preview android studio https://developer.android.com/studio/preview/install-preview
1. Configure desktop head unit https://developer.android.com/training/cars/testing#configure-dhu
1. cd $(ANDROID_HOME)/extras/google/auto/ && ./desktop-head-unit -c default_1080p.ini

Reply:
I changed these margins, so that the widgets are better aligned with the rest of the buttons. Also Maps example use the same values: https://github.com/mapbox/mapbox-maps-android/blob/main/android-auto-app/src/main/java/com/mapbox/maps/testapp/auto/car/CarMapWidgets.kt.

Reply:
By the way, I changed these margins, so that the widgets are better aligned with the rest of the buttons. Like in the screenshot below you can see that compass widget now has the same margin as the buttons on the right side. 
<img width="912" alt="Screenshot 2022-07-20 at 21 58 30" src="https://user-images.githubusercontent.com/2395284/180061244-a80c3cef-f72e-419c-a980-5cc8ee15ff0d.png">